app/_hub/moesif/kong-plugin-moesif/index.md

```diff
@@ -116,18 +132,34 @@ params:
       required: false
       default: "`{}`"
       description: An array of response body fields to mask.
+    - name: batch_size
+      required: false
+      default: "`200`"
+      description: Maximum batch size when sending to Moesif.
+    - name: event_queue_size
+      required: false
+      default: "`5000`"
+      description: Maximum number of events to hold in queue before sending to Moesif. In case of network issues when not able to connect/send event to Moesif, skips adding new to event to queue to prevent memory overflow.
+    - name: disable_gzip_payload_decompression
+      required: false
+      default: "`false`"
+      description: If set to true, will disable decompressing body in Kong.
+    - name: max_callback_time_spent
+      required: false
+      default: "`2000`"
+      description: Limiter on how much time to send events to Moesif per worker cycle.
+    - name: request_max_body_size_limit
+      required: false
+      default: "`100000`"
+      description: Maximum request body size in bytes to log in Moesif.
+    - name: response_max_body_size_limit
+      required: false
+      default: "`100000`"
+      description: Maximum response body size in bytes to log in Moesif.
     - name: debug
       required: false
       default: false
       description: An option if set to true, prints internal log messages for debugging integration issues.
-    - name: user_id_header
```

Comment:
Same as `timeout` option: `user_id_header` and `company_id_header` are still in the plugin's schema file. Are they deprecated, and if so, which options replace them?